src/js/components/CheckBoxGroup/CheckBoxGroup.js

```diff
@@ -9,12 +9,15 @@ import { useThemeValue } from '../../utils/useThemeValue';
 const CheckBoxGroup = forwardRef(
   (
     {
+      'aria-label': ariaLabelProp,
```

Comment:
Can you explain why aria-label takes precedence?

Reply:
I was thinking that if someone intentionally passes an aria-label then we should announce that instead of the FormField label

Reply:
From this MDN docs -- aria-labelledby should take precedence: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-labelledby

"aria-labelledby takes precedence over all other methods of providing an accessible name, including aria-label, [<label>](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/label), and the element's inner text."

Can you confirm we're not overriding this default behavior?